projects/simplicial-complex-operators/src/simplicial-complex-operators.cpp

```diff
@@ -120,8 +156,32 @@ Vector<size_t> SimplicialComplexOperators::buildFaceVector(const MeshSubset& sub
  */
 MeshSubset SimplicialComplexOperators::star(const MeshSubset& subset) const {
 
-    // TODO
-    return subset; // placeholder
+    auto starVertices = buildVertexVector(subset);   
+    auto starEdges = buildEdgeVector(subset);   
+    auto starFaces = buildFaceVector(subset);   
+
+    auto vertexToEdge = buildVertexEdgeAdjacencyMatrix();
```

Comment:
We have matrix A0 and A1 in constants. But I found out it too late and did the same)
https://github.com/Sergio-shu/ddg-exercises/blob/e297ae6f72a8dc4b255ac9800ef36f853265f209/projects/simplicial-complex-operators/include/simplicial-complex-operators.h#L16